### PR TITLE
Add OP RetroPGF Attestation Decoding

### PR DESCRIPTION
**Pull request by MSilb7**
Brief comments on the purpose of your changes:

**For Dune Engine V2**
- Create a RetroPGF folder.
- Add RetroPGF Round 2 decoding
- Prep the folder for backfilling round 1 and adding future rounds
- Add to labels
- Also cleaning up some frontend post-hooks for op_token (my b)

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/op/retropgf/optimism/op_retropgf_optimism_voters.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    block_date,
+    voter,
+    issuer,
+    can_vote
+
+    FROM {{ source[1] }}
+    {% if not loop.last %}
+    UNION ALL
+    {% endif %}
+    {% endfor %}
```
Comment (Hosuke) [inline models/op/retropgf/optimism/op_retropgf_optimism_voters.sql:29]: Great use of jinja loop

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @MSilb7 

[Sample labels_op_retropgf](https://dune.com/queries/2335584)
[Sample op_retropgf_optimism_round2_voters](https://dune.com/queries/2335582?d=1)

File: models/op/retropgf/optimism/op_retropgf_optimism_voters.sql (hunk 1 above)
Comment (MSilb7) [reply models/op/retropgf/optimism/op_retropgf_optimism_voters.sql:29]: copied this from another spell haha

**Review by jeff-dude — APPROVED** [review]
pushed a few final tweaks:
- static tags on tables with end dates, so prod doesn't run daily
- removed directory for `round2`, as the main intention for directories is differing schema names, these only changes table/view names

**Pull request merged**